addons/bus/static/tests/helpers/mock_python_environment.js

```diff
@@ -112,6 +120,9 @@ export const pyEnvTarget = {
         this._authenticate(user);
         this.cookie.set("authenticated_user_sid", this.cookie.get("sid"));
     },
+    getUserSettings() {
+        return {}; // dummy method to be overriden
+    },
```

Comment:
Note: only mail dependent tests need this value, and mock of method are defined in mail